Mathlib/Analysis/InnerProductSpace/Positive.lean

```diff
@@ -33,20 +35,128 @@ of requiring self adjointness in the definition.
 Positive operator
 -/
 
-
-open InnerProductSpace RCLike ContinuousLinearMap
+open InnerProductSpace RCLike LinearMap ContinuousLinearMap
 
 open scoped InnerProduct ComplexConjugate
 
-namespace ContinuousLinearMap
-
 variable {𝕜 E F : Type*} [RCLike 𝕜]
 variable [NormedAddCommGroup E] [NormedAddCommGroup F]
 variable [InnerProductSpace 𝕜 E] [InnerProductSpace 𝕜 F]
-variable [CompleteSpace E] [CompleteSpace F]
 
 local notation "⟪" x ", " y "⟫" => inner 𝕜 x y
 
+namespace LinearMap
+
+variable [FiniteDimensional 𝕜 E] [FiniteDimensional 𝕜 F]
+
+/-- A linear map `T` of a Hilbert space is **positive** if it is self adjoint and
+  `∀ x, 0 ≤ re ⟪T x, x⟫`. -/
+def IsPositive (T : E →ₗ[𝕜] E) : Prop :=
+  IsSelfAdjoint T ∧ ∀ x, 0 ≤ re ⟪T x, x⟫
+
+theorem IsPositive.isSelfAdjoint {T : E →ₗ[𝕜] E} (hT : IsPositive T) : IsSelfAdjoint T :=
+  hT.1
+
+theorem IsPositive.isSymmetric {T : E →ₗ[𝕜] E} (hT : IsPositive T) : IsSymmetric T :=
+  (isSymmetric_iff_isSelfAdjoint T).mpr hT.isSelfAdjoint
+
+theorem IsPositive.re_inner_nonneg_left {T : E →ₗ[𝕜] E} (hT : IsPositive T) (x : E) :
+    0 ≤ re ⟪T x, x⟫ :=
+  hT.2 x
+
+theorem IsPositive.re_inner_nonneg_right {T : E →ₗ[𝕜] E} (hT : IsPositive T) (x : E) :
+    0 ≤ re ⟪x, T x⟫ := by rw [inner_re_symm]; exact hT.re_inner_nonneg_left x
+
+open ComplexOrder in
+theorem isPositive_iff_isSelfAdjoint_and_inner_nonneg_left (T : E →ₗ[𝕜] E) :
+    IsPositive T ↔ IsSelfAdjoint T ∧ ∀ x, 0 ≤ ⟪T x, x⟫ := by
```

Comment:
```suggestion
open ComplexOrder in
theorem isPositive_iff (T : E →ₗ[𝕜] E) :
    IsPositive T ↔ IsSelfAdjoint T ∧ ∀ x, 0 ≤ ⟪T x, x⟫ := by
```
How about just calling it like this? This is what we'll want to use 99% of the time I suspect. (And the same for the continuous linear map version below.)